setup.py

```diff
@@ -1147,7 +1147,7 @@ def main():
         "filelock",
         "typing-extensions>=4.8.0",
         'setuptools ; python_version >= "3.12"',
-        'sympy==1.13.1 ; python_version >= "3.9"',
+        'sympy>=1.13.3 ; python_version >= "3.9"',
```

Comment:
Why not match requirements.txt here?

Reply:
If user wants to shoot themselves in the foot we should not stop them.